wynnset/tapestry-wp
# Do not return locked node content until node is actually unlocked

Closes #485

* Moves all locking functionality to backend
* Send node content only if the nodes are unlocked and accessible

**Testing**

* As a non-contributor, open a Tapestry with some locked nodes
* Verify that the locked nodes do not have their content through either the console or looking at the Vue extension and checking the Vuex store
  * Empty content in this case means the `quiz` property is an empty array and `typeData` only contains `progress` information
* Unlock the node
* Verify that the node unlocks without a need to refresh
* Open that node and verify the content looks correct
